README.md

```diff
@@ -111,6 +111,7 @@ BuildKit is used by the following projects:
 -   [Depot](https://depot.dev)
 -   [Namespace](https://namespace.so)
 -   [Unikraft](https://unikraft.org)
+-   [DevZero](https://devzero.io)
```

Comment:
Do you have a link describing how buildkit is being used?

Reply:
@tonistiigi sorry for the delay in responding
we have this on our public docs page that goes into some details: https://www.devzero.io/docs/recipes/build-process
let me know if you need anything else!